cli-feed-v4.json

```diff
@@ -24443,6 +24453,380 @@
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~4",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+              "FUNCTIONS_INPROC_NET8_ENABLED": "1"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v8.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET|8.0"
+            }
+          }
+        }
+      },
+      "coreTools": [
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.linux-x64.4.0.5780.zip",
+          "sha2": "4fbeac42a76c7265b64dd0967f398671ccfd739f7e906c40b22f87432f126d52",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.osx-x64.4.0.5780.zip",
+          "sha2": "fa9327e1b075f08a70d4b49e13837ed335fa98cf80e0ce26e9778154e92e5ac4",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "arm64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.osx-arm64.4.0.5780.zip",
+          "sha2": "77f6e29b957bc0d64ef76dea3325c686a16a4316fed6e616b61fc9e74e70690d",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-x64.4.0.5780_net8.0.zip",
+          "sha2": "d139149f897a0a39a1c960e71e4bdd2a475e4fda4e1cba14c684c2b421aa1935",
+          "size": "minified",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.win-x86.4.0.5780.zip",
+          "sha2": "dc9917152c6becf8919e864933a9d137b52cdf0cc51ac5c1d6776e5fa0aae333",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-x86.4.0.5780_net8.0.zip",
+          "sha2": "ed3234722696a8f621e3179a644955d2399591d61e25307486ab6ecc8165f7aa",
+          "size": "minified",
+          "default": "true"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "arm64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-arm64.4.0.5780_net8.0.zip",
+          "sha2": "a31bd555992af47cebefa6e0e82afbe3831ab1df5cf8b525774fd9e17d4004aa",
+          "size": "minified",
+          "default": "false"
+        }
+      ]
```

Comment:
Why are these versions different from each other? They should all have the `_net8.0` suffix shouldn't they?

Reply:
Had an offline discussion about this with @fabiocav. We are not blocking on it at the moment. VS only uses windows and mac os versions from the feed.

Reply:
My question earlier was about other tooling. Is there a reason why this wouldn't be updated? If we don't have the builds/targets, let's remove them from here. It's better to fail to find a match than match to something that will fail at runtime (with confusing messages)

Reply:
Let's remove any of the entries without a matching version of Core Tools

Reply:
We are going to generate net8 entries for all variations. Have updated the PR to make all entries _net8